service/assessment/assessment.go

```diff
@@ -171,24 +216,145 @@ func (s Service) handleEvidence(evidence *evidence.Evidence, resourceId string)
 		// Just a little hack to quickly enable multiple results per resource
 		s.results[fmt.Sprintf("%s-%d", resourceId, i)] = result
 
-		s.informHooks(result, nil)
+		// TODO(all): Currently, we send result via hook. But I think it would be cleaner to do it straight here.
+		go s.informHooks(result, nil)
 	}
 
 	return nil
 }
 
+// convertTargetValue converts the given target_value in a format accepted by protobuf
+// Note: Therefore, we have to be specific about the format of target values in the REGO code
+func convertTargetValue(value interface{}) (convertedTargetValue *structpb.Value, err error) {
```

Comment:
This function replicates behaviour of `structpb.NewValue()` why? Not necessary in my opinion.

Reply:
We do it with `json` methods `Marshal` and `Unmarshal` now